src/pages/Strategies/hooks/use-get-strategy-available-amounts.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  useGetLoanAvailableAmounts,
+  UseGetLoanAvailableAmountsResult
+} from '@/hooks/api/loans/use-get-loan-available-amounts';
+
+import { StrategyFormType, StrategyType } from '../types';
+import { StrategyData } from './use-get-strategies';
+import { StrategyPositionData } from './use-get-strategy-position';
+
+type useGetStrategyAvailableAmountsResult = UseGetLoanAvailableAmountsResult;
+
+const useGetStrategyAvailableAmounts = (
+  type: StrategyFormType,
+  strategy: StrategyData,
+  position?: StrategyPositionData
+): useGetStrategyAvailableAmountsResult => {
+  const result = useGetLoanAvailableAmounts(
```

Comment:
I would rename this to more descriptive `loanAvailableAmounts`, there will probably be multiple different data points that impact the `result`.